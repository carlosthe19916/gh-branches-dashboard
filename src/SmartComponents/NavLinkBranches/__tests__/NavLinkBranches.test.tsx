import React from "react";
import { shallow } from "enzyme";
import {
  NavLinkBranches,
  NavLinkBranchesStateToProps
} from "../NavLinkBranches";

it("Test snapshot :: with ctxRepository", () => {
  const props: NavLinkBranchesStateToProps = {
    ctxRepository: {
      full_name: "carlosthe19916/gh-branches-dashboard"
    }
  };

  const wrapper = shallow(<NavLinkBranches {...props} />);
  expect(wrapper).toMatchSnapshot();
});

it("Test snapshot :: without ctxRepository", () => {
  const props: NavLinkBranchesStateToProps = {
    ctxRepository: undefined
  };

  const wrapper = shallow(<NavLinkBranches {...props} />);
  expect(wrapper).toMatchSnapshot();
});
