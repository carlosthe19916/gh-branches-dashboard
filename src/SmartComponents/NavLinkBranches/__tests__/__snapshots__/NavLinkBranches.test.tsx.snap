// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot :: with ctxRepository 1`] = `
<NavLink
  activeClassName="pf-m-current"
  className="pf-c-nav__link"
  to="/monitor/carlosthe19916/gh-branches-dashboard/branches"
>
  Branches
</NavLink>
`;

exports[`Test snapshot :: without ctxRepository 1`] = `<Fragment />`;
